instructors.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: page
+title: Programming with Python
+subtitle: Instructor's Guide
+---
+## Legend
+
+FIXME: story behind running example.
+
+## Overall
+
+This lesson is written as an introduction to Python,
+but its real purpose is to introduce the single most important idea in programming:
+how to solve problems by building functions,
+each of which can fit in a programmer's working memory.
+In order to teach that,
+we must teach people a little about
+the mechanics of manipulating data with lists and file I/O
+so that their functions can do things they actually care about.
+Our teaching order tries to show practical uses of every idea as soon as it is introduced;
+instructors should resist the temptation to explain
+the "other 90%" of the language
+as well.
+
+The secondary goal of this lesson is to give them a usable mental model of how programs run
+(what computer science educators call a [notional machine](reference.html#notional-machine)
+so that they can debug things when they go wrong.
+In particular,
+they must understand how function call stacks work.
```

Comment:
Since #23 this has been moved to the supplementary material. Should that be reflected here?